### ISSUE TITLE
 Discuss potential further change to induced charge interactive alert when rubbing against wall

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#252]
Based on recent improvements to the induced charge description (see related issue #249), there is a new interaction alert for rubbing at the wall.

At the PhET/IDRC A11y meeting Feb 21, 2017, people agreed to a new wall rubbing interactive alert.

This can be implemented as is, however, I had a couple more design questions (see below) that could potentially shorten this rather long alert. I'd like to run these questions by @ariel-phet, @emily-phet, and @jessegreenberg. 

**Here's the design of the new interactive alert:**
- [At [[STRING:Location]] wall. No transfer of charge.] [+ Induced charge description]. [[STRING:Balloon name]] [+ Balloon's relative charge description (no capitalization)]. [+ Wall’s charge description].

**Example when rubbing wall with a neutral balloon** 
- Down. At [[lower]] wall. No transfer of charge. In [[lower]] wall, no change in charges. [[Yellow balloon]] has [[no]] more negative charges than positive charges. Wall has many pairs of negative and positive charges.

**Example when rubbing wall with a charged balloon** 
- Up. At [[upper]] wall. No transfer of charge. Negative charges in [[upper]] wall move away from [[yellow balloon]] [[quite a lot]]. Positive charges do not move. [[Yellow balloon]] has [[many more]] negative charges than positive charges. Wall has many pairs of negative and positive charges.

**Design Questions:** 
1. Is the order optimal? Do you think it would sound better with the balloon's charge last?
2. How do you feel about truncating the alert after the first rub by dropping the charge information for the wall and balloon, or perhaps shortening the charge information by using the net charge descriptions. Again, this would be after the first rub. 

**Example successive rubs on wall with a neutral balloon** 
- Down. At [[lower]] wall. No transfer of charge. In [[lower]] wall, no change in charges.

OR

- Down. At [[lower]] wall. No transfer of charge. In [[lower]] wall, no change in charges. [[Yellow balloon]] has neutral net charge. Wall has neutral net charge.

**Successive rubs on wall with a charged balloon** 
- At [[upper]] wall. No transfer of charge. Negative charges in [[upper]] wall move away from [[yellow balloon]] [[quite a lot]]. Positive charges do not move.

OR

- At [[upper]] wall. No transfer of charge. Negative charges in [[upper]] wall move away from [[yellow balloon]] [[quite a lot]]. Positive charges do not move. [[Yellow balloon]] has negative net charge. Wall has neutral net charge.

**Comment by terracoda** [comment]
I've add a comparison of two mini cognitive walk-throughs in the [google sheet BASE: CogWTs: New Grid](https://docs.google.com/spreadsheets/d/1fZq21j65jfJhcsA22pasuBA00kM-cGWxsWDsogwhC64/edit#gid=0)

**Comment by ariel-phet** [comment]
@terracoda the order seems good to me and fairly optimal.

As for the second question, that seems more of an interview/user question. As long as the user can keep track I would be fine with truncating (it seems reasonable), but not really a question I feel I am in a position to answer.

**Comment by terracoda** [comment]
@ariel-phet, thanks for your feedback. I agree truncation questions (of which I have several) should be addressed after next round of interviews.

**Comment by terracoda** [comment]
Decision, we will stick with the current new design mentioned in https://github.com/phetsims/balloons-and-static-electricity/issues/252#issue-211067818, until we have more interview data suggesting otherwise.

I am un-assigning @emily-phet and leaving myself and @jessegreenberg assigned until implementation.

**Comment by terracoda** [comment]
Actually closing this issue, as we are sticking with the full description as described in the original related issue #249.

**Comment by terracoda** [comment]
Reopening, as discussions with screen reader user has confirmed that it is ok, to drop the non-changing balloon and wall visual charge information after the initial rub (or after a grab at the wall). The visual charge information does not change. The information is also available on grab, and in the PDOM, in case the user needs to review it.

**Truncated interactive wall rubbing alert**
- [At [[STRING:Location]] wall. No transfer of charge.] [+ Induced charge description]. [[STRING:Balloon name]].

**Example when rubbing wall with a neutral balloon**
- Down. At [[lower]] wall. No transfer of charge. In [[lower]] wall, no change in charges.

**Example when rubbing wall with a charged balloon**
- Up. At [[upper]] wall. No transfer of charge. Negative charges in [[upper]] wall move away from [[yellow balloon]] [[quite a lot]]. Positive charges do not move.

**Comment by terracoda** [comment]
@jessegreenberg, I think the design docs is up to date with more examples. I am working on making sure they are clean

**Comment by terracoda** [comment]
Induced charge descriptions and alerts are now implemented. I am closing this issue.
